etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/utils/security/HashHelper.java

```diff
@@ -0,0 +1,25 @@
+package gov.hhs.cdc.trustedintermediary.etor.utils.security;
+
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.HexFormat;
+import javax.inject.Inject;
+
+public class HashHelper implements SecureHash {
+    @Inject Logger logger;
+
+    @Override
+    public <T> String generateHash(T input) {
```

Comment:
I don't think we need to use generics here.  I.e. the `T` stuff.  You could probably just use `Object` since we aren't really using the functionality of generics in this method.

Reply:
I'll defer to that